canonical/lxd-ui
# simpler one click edit mode for edit forms

## Done

- simpler one click edit mode for edit forms on instances, profiles networks, storage pools, storage volumes and projects

Fixes WD-14393

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - check edit and creation of instances, profiles networks, storage pools, storage volumes and projects
